src/Bootstrap.php

```diff
@@ -29,17 +33,27 @@ public static function getApplication(): Application
         return self::$application;
     }
 
-    private static function buildApplication(): Application
+    private static function getApplicationConfig()
     {
-        $mergedApplicationConfig = ArrayUtils::merge(
-            require 'config/application.config.php',
-            require 'config/development.config.php'
-        );
+        $applicationConfig = require 'config/application.config.php';
+
+        if (file_exists('config/development.config.php')) {
+            $applicationConfig = ArrayUtils::merge($applicationConfig, require 'config/development.config.php');
+        }
+
+        if (file_exists('config/test.config.php')) {
```

Comment:
Good for now but you could also use https://github.com/zendframework/zend-config-aggregator for this.